src/planner/subquery/flatten_dependent_join.cpp

```diff
@@ -282,6 +282,23 @@ unique_ptr<LogicalOperator> FlattenDependentJoins::PushDownDependentJoinInternal
 			return move(plan->children[0]);
 		}
 	}
+	case LogicalOperatorType::LOGICAL_LIMIT_PERCENT: {
+		auto &limit = (LogicalLimitPercent &)*plan;
```

Comment:
This code is not run. Could you add a test case with a correlated subquery that uses `LIMIT%` so that all the code here is being run? Alternatively you can remove this code and just emit an error message in this case.

Reply:
Okay, I will fixt it